setup.py

```diff
@@ -20,12 +20,11 @@
               "jsmol-bokeh-extension~=0.2.1",
               "pandas~=0.24.2",
               "requests~=2.21.0",
-              "panel~=0.6.4",
-              "param~=1.9.2",
+              "panel~=0.8",
+              "param~=1.9.3",
               "jupyter",
               "graphviz~=0.13",
-          ],
-          extras_require={
-              "pre-commit":
-              ["pre-commit==1.17.0", "prospector==1.2.0", "pylint==2.4.0"]
-          })
+              "pre-commit",
+              "yapf==0.28.0",
+              "pylint>=2.4.0"
```

Comment:
usually a good idea to fix versions for the linter stuff, since they come up with new rules every now and then